UI/src/app/pages/home/home.component.scss

```diff
@@ -23,13 +24,14 @@
 }
 
 .carousel-img>img {
-  min-width: 100%;
   margin: 0 auto;
 }
 
 .carousel-img {
   min-width: 100%;
```

Comment:
this parameter causes that there will be no change of the image size when the page is resized, but what fixes the problem with images we had, are the max-width and max-height properties